docs/conventions/print.md

```diff
@@ -0,0 +1,232 @@
+# How to handle print css rules in ECL
+
+## Adopted solution
+
+- Print CSS and screen CSS are kept into separated files
+    ```
+    <head>
+      ...
+      <link rel="stylesheet" type="text/css" href="ecl-<preset>.css"  media="screen">
+      <link rel="stylesheet" type="text/css" href="ecl-<preset>-print.css" media="print">
+    </head>
+    ```
+
+- No global reset CSS for print
+
+- Print CSS is applied per component, using component's classes. Only needed rules are reported, to keep print CSS as small as possible.
+    ```
+    (screen css)
+    .my-component {
+      background-color: <yellow>;
+      color: <grey>;
+      display: flex;
+      font: <font-m>;
+      width: 100%;
+    }
+    ```
+    ```
+    (print css)
+    .my-component {
+      color: black;
+      font: <font-print-m>;
+      width: 100%;
+    }
+    ```
+    
+- Design tokens are added for print display (typography, spacing, ...)
+
+Below are the discussed options, for the record
+
+
+## File structure
+
+Mainly anwser this question: should we group all CSS in one file or split it in several files?
+
+### Option 1: One CSS for everything
+
+It means that screen and print CSS rules would be in a single SCSS file.
+Print CSS rules would be put in a `@media print` section.
+All SCSS files are merged in a `ecl-<preset>.css` file.
+
+Example:
+
+```
+(my-component.scss)
+.my-component {
+  <component-css-rules>
+}
+
+@media print {
+  .my-component {
+    <component-print-css-rules>
+  }
+}
+```
+
+```
+<link rel="stylesheet" type="text/css" href="ecl-<preset>.css">
+```
+
+### Option 2: Two separated css
+
+It means adding an extra CSS file for print only
+Each component would have 2 SCSS files (one for print one for screen)
+Screen files are merged into a `ecl-<preset>.css` file; print files are merged into `ecl-<preset>-print.css`.
+
+Example:
+
+```
+(my-component.scss)
+.my-component {
+  <component-css-rules>
+}
+```
+
+```
+(my-component-print.scss)
+.my-component {
+  <component-print-css-rules>
+}
+```
+
+```
+<link rel="stylesheet" type="text/css" href="ecl-<preset>.css" media="screen">
+<link rel="stylesheet" type="text/css" href="ecl-<preset>-print.css" media="print">
+```
+
+## CSS structure
+
+Independently to file structure, the goal here is to discuss how we write CSS rules for print.
+
+### Option 1: Separate screen and print
+
+It means having all css rules related to screen display grouped together, and all print rules grouped together somewhere else.
+
+Example:
+
+```
+(media screen)
+.my-component {
+  color: <grey>;
+  display: flex;
+  font: <font-m>;
+  width: 100%;
+}
+
+(media print)
+.my-component {
+  color: black;
+  display: flex;
+  font: <font-print-m>;
+  width: 100%;
+}
+```
+
+Pros:
+
+- Clear separation of styles
+
+Cons:
+
+- Have to specify all css rules in print (duplication)
```

Comment:
it's only duplicated if we want the same result... most of the time, I guess we won't want the same output :)